metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -165,6 +171,48 @@ def extract_dataset_schema(
         )
         return [schema_mcp]
 
+    def make_fine_grained_lineage_class(
+        self, lineage: resolver.Lineage, dataset_urn: str
+    ) -> List[FineGrainedLineage]:
+        fine_grained_lineages: List[FineGrainedLineage] = []
+
+        if (
+            self.__config.extract_column_level_lineage is False
+            and self.__config.extract_lineage is False
```

Comment:
what are we trying to do here ? Should this be OR ?

Reply:
done